servers/src/main/java/tachyon/master/MasterBase.java

```diff
@@ -41,6 +44,8 @@
 public abstract class MasterBase implements Master {
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
 
+  private static final long SHUTDOWN_TIMEOUT_MS = 10000;
```

Comment:
should this be a config parameter? Also, when would a long timeout be desired? It seems like if you want to shutdown a master, you want it to be done as soon as possible?